### PR TITLE
[DST-424]: setup codehike

### PR DESCRIPTION
**Pull request by aromko**

### CODE REVIEW

**Comment by aromko** [comment]
I've had problems with jest and mdx. I found a workaround when using vitest. Maybe for now it's good enough.

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
+function App() {
   return (
-    <>
-      <h1>Reference App</h1>
-      <h1 className="text-3xl font-bold underline">Hello World!</h1>
-    </>
+    <article>
+      <div>
+        <Example />
```
Comment (sarahgm) [inline src/App.tsx:10]: should there be a navigation for the reference apps oder how is this handled if there are more?

Comment (aromko) [reply src/App.tsx:10]: Yeah that's missing atm. Currently I write a ticket and will came up with a proposal :-)

Comment (aromko) [reply src/App.tsx:10]: See https://reservix.atlassian.net/browse/DST-432

**Review by sarahgm — APPROVED** [review]

**Pull request merged**